setup.cfg

```diff
@@ -30,6 +30,6 @@ disable=abstract-method, # raises false positives in beam
 
 [FORMAT]
 # Allow some single and two letter variables, but keep overall invalid-name rule
-good-names=i,j,k,f,ex,e,_,p,ip,cm
+good-names=i,j,k,f,ex,e,_,p,ip,cm,censoredplanet-analysis
```

Comment:
add an explanatory comment here similar to https://github.com/censoredplanet/censoredplanet-analysis/pull/113/commits/dfeefcfb2c48527aec9132a9b9c7309fa86228a9